agent/mapcall/src/main/resources/default-replacements.txt

```diff
@@ -0,0 +1,35 @@
+// Default replacement file
+//
+// Notes:
+// 1. This replacement file only uses package replacement.  It requires that all desired
+//    replacements exist in the replacement (e.g., second) package, and that there is a
+//    corresponding method in the original package.
+// 2. Experiments showed that creating a blanket null/zero-replacement of methods/constructors for
+//    the AWT/Swing classes breaks the behavior so drastically that methods under test are affected,
+//    and creates uninteresting or failing tests b/c of use of null or zero values.
+
+// Apply default mock for System.exit that throws randoop.SystemExitCalledError.
+java.lang randoop.mock.java.lang
+
+// Apply mocks for AWT/Swing to (attempt to) avoid the following:
```

Comment:
This is useful information, but I would expect it to be written together with the code that it documents, such as in a README file in `agent/mapcall/src/main/java/randoop/mock`.  A cross-reference from here to that file would be useful.  But if the information appears only here, then I expect that it will be very hard for people to find it.